src/encoder.rs

```diff
@@ -418,14 +418,15 @@ impl<T: Pixel> FrameState<T> {
     )
   }
 
+  /// Does not create qres or hres versions of `frame`
```

Comment:
Since this function behaves very specifically, I think it would be good to add a note of why this function exists.